docs/tutorial/extensions.md

```diff
@@ -45,6 +45,49 @@ The wrapped function:
 If any of these constraints are violated, the outcome is undefined.
 {% endhint %}
 
+## fhe.multivariate(function)
+
+Allows you to wrap any multivariate function into a table lookup:
+
+```python
+import numpy as np
+from concrete import fhe
+
+def keep(value, keep):
```

Comment:
can the function and parameter have different name ?
hint: value_or_zero(value, make_it_zero)

Reply:
It's this now:
```python
def value_if_condition_else_zero(value, condition):
    return value if condition else np.zeros_like(value, dtype=np.int64)
```